README.md

```diff
@@ -2,3 +2,19 @@
 
 [![Build Status](https://travis-ci.org/hmcts/bulk-scan-orchestrator.svg?branch=master)](https://travis-ci.org/hmcts/bulk-scan-orchestrator)
 [![Codacy Badge](https://api.codacy.com/project/badge/Grade/e9272daf4b714e4f95280916e763b6b2)](https://www.codacy.com/app/HMCTS/bulk-scan-orchestrator)
+
+## Running end to end using docker
+- Run up the docker environment from bulk-scan-shared-infrastructure
+- You will have to setup these environment variables on either your
+run configuration or bash shell
+```$bash
+IDAM_USERS_SSCS_USERNAME = bulkscanorchestrator+systemupdate@gmail.com
+IDAM_USERS_SSCS_PASSWORD = Password12
+CORE_CASE_DATA_API_URL = http://localhost:4452
+```
+- add a case into using the docker ui
+- copy the case number from the UI (excluding the #) and place it into the example1.json#case_ref to reference the created case.
```

Comment:
excluding the # and -'s ?

Reply:
Can quite rememeber if the reference has - on the ui ?
Ill add that anyway.